tests/integration/operator/ginkgo/suite_test.go

```diff
@@ -0,0 +1,147 @@
+package controllers_test
+
+import (
+	"context"
+	"os"
+
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/utils/pointer"
+
+	//ctrl "sigs.k8s.io/controller-runtime"
```

Comment:
nit: Can you do a quick sweep for commented out code and remove lines like this?